src/pngchunk_int.cpp

```diff
@@ -162,6 +163,15 @@ namespace Exiv2 {
         }
         else if(type == iTXt_Chunk)
         {
+            int nullSeparators = 0;
```

Comment:
[suggestion] This can be also performed with [std::count](http://en.cppreference.com/w/cpp/algorithm/count).

Reply:
Yes, this is what I was thinking :). Now that we agreed on the solution I will use that algorithm.